src/Entity/Storage/EdgeEntityStorageBase.php

```diff
@@ -335,15 +335,10 @@ final protected function processLoadedEntities(?array $ids, array $sdk_entities)
   protected function invokeStorageLoadHook(array &$entities) {
     if (!empty($entities)) {
       // Call hook_entity_storage_load().
-      foreach ($this->moduleHandler()->getImplementations('entity_storage_load') as $module) {
-        $function = $module . '_entity_storage_load';
-        $function($entities, $this->entityTypeId);
-      }
+      $this->moduleHandler()->invokeAll('entity_storage_load', [$entities, $this->entityTypeId]);
```

Comment:
The array content should be split over multiple lines to avoid array column limitation(_80 column_) errors.